### DISCUSSION TITLE
kali-ios_v.1.1.0

### DISCUSSION THREAD

**Discussion by finlandhl** [finlandhl/kali-ios#8] [Announcements]
I’m pleased to announce the upcoming update for Kali iOS, bringing exciting new features and improvements to enhance your experience with Kali Linux on iOS. Here are the key details of this release:

1. Full Support for pip3 and python3: I have ensured that pip3 and python3 are fully functional in this update. You can now harness the power of Python and install packages using pip3 on your iOS device.

2. Introducing Links and Elinks Browsers: I have compiled and added Links and Elinks browsers to the Kali iOS environment. Now you can browse the web directly from your iOS device using these text-based browsers, providing you with more flexibility and control.

3. Renaming and Relocating Scripts: To improve organization and ease of use, I have renamed and relocated certain scripts in this update. This will make it more convenient for you to find and execute the scripts you need within the Kali iOS environment.

4. New 'debinst' Command: I have introduced a new command called 'debinst' in this update. You can now use 'debinst' to install any .deb archives as an alternative to apt which refuses to install packages for some mysterious reasons as well as aptitude. Remember to place the .deb archive in the /var/cache/apt/archives directory and specify it when prompted by the program.

5. 'bugfix' Command: I have also added a 'bugfix' command to address common issues within the Kali iOS environment. Simply type 'bugfix' to initiate the program and resolve any known issues automatically 

6. Caution Regarding libc6 Upgrade: Please note that upgrading libc6 (glibc 2.28) is not recommended unless you have the necessary debugging skills. Proceed with caution and ensure compatibility by downloading and installing debian packages that are compatible with your installed version of libc6.

7. apt-get Limitation: It's important to be aware that apt-get is still not functioning optimally in this update. I’m actively working to address this and will provide further updates in future releases. Anyone who was able to fix it is invited to share with the community.

8. Enhanced Library Support: I have installed numerous libraries and missing dependencies to make it easier for you to compile programs within the Kali iOS environment. Enjoy a smoother experience when compiling and running your software (recommend iPhone 12 and later for compilation tasks)

9. Fine-Tuned Screen Elements: I have fine-tuned the elements displayed on the screen for better layout and readability. To optimize the display, I  recommend reducing the font size to 9 in the ISH Shell Appearance Settings.

Thank you for your continuous support and valuable feedback. I hope that these new additions and improvements enhance your usage of Kali Linux on your iOS device. Stay tuned for more exciting updates in the future!

If you have any questions or need further assistance, please don't hesitate to reach out.

<hr /><em>This discussion was created from the release <a href='https://github.com/finlandhl/kali-ios/releases/tag/kali-ios_v1.1.0'>kali-ios_v.1.1.0</a>.</em>

**Comment by Luci4n0z** [comment]
I don't know now if it's right or not. Because now I can't enter ish, it always closes itself.

**Reply by huayem** [reply]
Open filesystem,booting to kali-ios. Don't touch ish app until restart your phone.

**Comment by n0r4c** [comment]
Chave de senha do Mega link? @finlandhl
